src/btree/bt_page.c

```diff
@@ -36,18 +36,18 @@ __wt_page_block_meta_assign(WT_SESSION_IMPL *session, WT_PAGE_BLOCK_META *meta)
      * some IDs for now.
      */
     page_id = __wt_atomic_fetch_add64(&btree->next_page_id, 1);
+    /* TODO: need to fix checkpoint id assignment for eviction during a checkpoint. */
```

Comment:
As mentioned in the comment above, we cannot do this given the current architecture.